tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -40,3 +109,40 @@ def test_kernel_attributes(self):
         assert self.kern.num_regs > 0
         assert self.kern.max_threads_per_block > 0
         assert self.kern.shared_size_bytes == 0
+
+    def test_module(self):
+        module = self.mod2
+        ker_sum = module.get_function('test_sum')
+        ker_times = module.get_function('test_multiply')
+
+        N = 10
+        x1 = cupy.arange(N**2, dtype=cupy.float32).reshape(N, N)
+        x2 = cupy.ones((N, N), dtype=cupy.float32)
+        y = cupy.zeros((N, N), dtype=cupy.float32)
+
+        ker_sum((N,), (N,), (x1, x2, y, N**2))
+        assert cupy.allclose(y, x1 + x2)
+
+        ker_times((N,), (N,), (x1, x2, y, N**2))
+        assert cupy.allclose(y, x1 * x2)
+
+    def test_compiler_flag(self):
+        module = self.mod3
```

Comment:
Just a nitpick,
Since the only thing that changes with this function and the above one is the module,
can we refactor the calculations and checks into a single function?

Reply:
OK. Take a look at e05d0fc.

Reply:
awesome!